doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,42 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the networking core image
+******************************************************
+
+In addition to the recovery of the application core image, also the networking core image can be recovered.
+You can use this feature if you build nRF53 MCUboot with one of the following options:
```

Comment:
```suggestion
When you build MCUboot for the nRF5340 DK, you can use this feature with one of the following options:
```

Reply:
Please apply this

Reply:
@FrancescoSer These options might be available for both nRF5340DK and thingy53 (or any else nRF53 board if user can meet dependencies).

Reply:
```suggestion
When you build MCUboot for the nRF5340 DK or the Thingy:53, you can use this feature with one of the following options:
```

Ok so let's edit this a bit for now, and we will see to move the content in the thingy:53 guide after this PR gets merged